src/components/Integrations/Shopify/Shopify.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import Loading from '../../Loading/Loading';
+import { InjectAppServices } from '../../../services/pure-di';
+
+const Shopify = ({ dependencies: { shopifyClient } }) => {
+  const [shops, setShops] = useState([]);
+  const [isConnected, setIsConnected] = useState(false);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const getData = async () => {
+      const result = await shopifyClient.getShopifyData();
+      if (result.success && result.value.length) {
+        setIsConnected(true);
+        setShops(result.value);
+      } else if (
+        !result.success &&
+        result.expectedError &&
+        result.expectedError.cannotConnectToAPI
+      ) {
+        setError('Error: No hemos podido conectar con la Api de Shopify, vuelve a intentar luego.');
+      } else if (!result.success) {
+        setError('Error: Error inesperado.');
+      }
+      setIsLoading(false);
+    };
+    getData();
+  }, [shopifyClient]);
+
+  return (
+    <>
+      <Helmet title={'Doppler | Shopify'} />
+      <section style={{ width: '100%', padding: '60px 30px' }}>
```

Comment:
its better use css class, what do you think?

Reply:
I think it's great, but it's something to fix in another PR. Then we see it.  ;)

Reply:
This is temporal, next I will be adding styles already merged into ui library, check https://cdn.fromdoppler.com/doppler-ui-library/latest/integrations.html

Reply:
can add html comment? to dont forget change

Reply:
@fcoronelmakingsense I can't seem to add html commented code in between. But I will remember, my next pr will be about applying styles in the library.